test/python/test_moe_grouped_matmul.py

```
@@ -278,3 +337,107 @@ def test_bf16_moe_grouped_matmul_bwd(cudnn_handle):
+        mode=cudnn.moe_grouped_matmul_mode.NONE,
+        compute_data_type=cudnn.data_type.FLOAT,
+        name="moe_grouped_matmul",
+    )
+    tensor_output.set_data_type(cudnn.data_type.BFLOAT16).set_output(True)
+
+    graph.validate()
+    graph.build_operation_graph()
+    graph.create_execution_plans([cudnn.heur_mode.A])
+    try:
+        graph.check_support()
+    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)